Moesif.Middleware/NetCore/MoesifMiddlewareNetCore.cs

```diff
@@ -162,10 +162,10 @@ private void ScheduleAppConfig()
                     }
                     catch (Exception e)
                     {
-                        _logger.LogError(e, "Error while scheduling appConfig job");
+                        _logger.LogError(e, "Error while updating appConfig");
                     }
-                    // wait for max 1 hour
-                    await Task.Delay(60 * 1000);
+                    
+                    configEvent.WaitOne(60 * 1000);
```

Comment:
this is one minute not an hour. Correct ?

Reply:
Correct.